From 5af603a441d2cda5e9cd847997954fd378058070 Mon Sep 17 00:00:00 2001
From: Pedro Palazon Candel <pedro@joyent.com>
Date: Wed, 8 Aug 2018 18:20:55 +0200
Subject: [PATCH] TRITON-636 trouble with timeouts in sdcadm health Reviewed
 by: Marsell Kukuljevic <marsell@joyent.com> Approved by: Marsell Kukuljevic
 <marsell@joyent.com>

---
 lib/sdcadm.js | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lib/sdcadm.js b/lib/sdcadm.js
index 8506a00..0b09436 100644
--- a/lib/sdcadm.js
+++ b/lib/sdcadm.js
@@ -3936,12 +3936,13 @@ SdcAdm.prototype.checkHealth = function checkHealth(opts, cb) {
         urConnection.exec({
             script: script,
             server_uuid: inst.server,
-            timeout: 5000
+            timeout: 10000
         }, function (err, result) {
             if (err) {
                 inst.health_errors = [ {
                     message: err.message
                 }];
+                self.log.error({error: err}, 'Ur execution error');
                 next(null, inst);
                 return;
             }
-- 
2.21.0

